addons/mail/static/src/core/common/thread.xml

```diff
@@ -3,7 +3,11 @@
 
 <t t-name="mail.Thread">
     <t t-if="env.inChatter" t-call="mail.Thread.jumpPresent"/> <!-- chatter has its own scrollable, this ensures proper sticky showing -->
-    <div class="o-mail-Thread position-relative flex-grow-1 d-flex flex-column overflow-auto" t-att-class="{ 'pb-4':  props.showJumpPresent and !state.showJumpPresent }" t-ref="messages" tabindex="-1">
+    <div class="o-mail-Thread position-relative flex-grow-1 d-flex flex-column overflow-auto"
+         t-att-class="{ 'pb-4':  props.showJumpPresent and !state.showJumpPresent,
+                        'px-3': !env.inChatter and !props.isInChatWindow }"
+         t-ref="messages"
+         tabindex="-1">
```

Comment:
Keep everything inline. I know XML sucks when lines are long, but it also sucks with multi-lines, e.g. it becomes hard to get a grasp of the shape of template at a glance due to weird alignment of siblings and closing of tag hard to spot.

The tradeoff we choose is keep everything inline. In some rare cases multi-line is fine, but here I wouldn't